data/ui/item_box.blp

```diff
@@ -43,13 +32,39 @@ template $ItemBox : Box {
     }
   }
 
-  Button {
-    tooltip-text: _("Remove");
+  MenuButton {
+    icon-name: "view-more-symbolic";
+    popover: popover_menu;
     styles ["flat"]
-    clicked => $delete();
-    Image {
-      hexpand: false;
-      icon-name: "edit-delete-symbolic";
-    }
   }
 }
+
+PopoverMenu popover_menu {
```

Comment:
I'm just wondering if it would be possible to use icons here and have move up and down horizontal to each other, wouldn't consider this a blocker however

Reply:
I think that would likely be an improvement, perhaps we can do something similar to the icon size in Nautilus:

![image](https://github.com/Sjoerd1993/Graphs/assets/68477016/1ec30fc6-2517-4e07-9005-48dbf661f301)

However, I cannot find any reference of how this would be done [in the documentations](https://docs.gtk.org/gtk3/class.PopoverMenu.html). It doesn't seem a GtkBox is supported as  a child to GtkPopoverMenu. Will try to see if I can find a solution later on, but we could do that as a separate PR perhaps.